packages/11ty/_includes/web-components/image-sequence/index.js

```diff
@@ -1,208 +1,129 @@
-
-      slot[name='images'] img.visible {
-        opacity: 1;
-        object-fit: contain;
-      }
-
-      slot[name='images'] img:not(.visible) {
-        opacity: 0;
-      }
-
-      slot[name='images'] img.fade-in {
-        animation: fade-in 0.25s 1 linear;          
-      }
-
-      @keyframes fade-in {
-        from {
-          opacity: 0;
-        }
-        to {
-          opacity: 1;
-        }
-      }
-
-      slot[name='placeholder-image'] img.loading {
-        opacity: 1;
-        filter: brightness(0.4);
-        animation: loading-image 1s linear infinite alternate;
-      }
-
-      @keyframes loading-image {
-        from {
-          filter: brightness(0.4);
-        }
-        to {
-          filter: brightness(0.2);
-        }
-      }
+class ImageSequence extends LitElement {
 
-      slot[name='placeholder-image'] img {
-        opacity: 0;
-        transition: opacity 0.25s linear;
-        object-fit: cover;
-      }`
+  static styles = [ imageSequenceStyles ]
 
   static properties = {
-    index: {
-      type: Number,
-    },
     didInteract: {
       type: Boolean,
       state: true,
     },
-    visible: {
-      type: Boolean,
-      state: true,
-    },
     images: {
       type: Array,
       state: true,
     },
+    imageUrls: {
+      attribute: 'image-urls',
+      type: String,
+    },
+    index: {
+      type: Number,
+    },
+    intrinsicHeight: {
+      type: Number,
+      state: true,
+    },
+    intrinsicWidth: {
+      type: Number,
+      state: true,
+    },
     rotateToIndex: {
       attribute: 'rotate-to-index',
       type: Number,        
     },
     transition: {
       type: Number,
+    },
+    visible: {
+      type: Boolean,
+      state: true,
     }
   }
 
-  get allImagesLoaded() {
-    return this.imagesLoaded === this.images.length
+  canvasRef = createRef()
+
+  get someImagesLoaded() {
+    return this.images.some( i => i !== null )
   }
 
-  // Fires when this component is visible
-  _loadImages() {
-    const imageElements = this.images.map( (img,i) => {
-      fetch(img).then( this.imagesLoaded++ )
-    })
+  /**
+   * @function #fetchImage
+   * @param url {string} - image URL to fetch
+   * @param seqIndex {Number} - index to store this image 
+   * @param draw {Boolean} - whether to draw after fetching
+   * 
+   * Fetches `url`, converts it into a blob and stores the image data, optionally drawing to the canvas
+   **/
+  #fetchImage(url,seqIndex,draw) {
+    const req = new Request(url)
+
+    fetch(req)
+      .then( (resp) => resp.blob() )
+      .then( (blob) => window.createImageBitmap(blob) )
+      .then( (bmp) => {
+        if (this.intrinsicHeight === 0) {
+          this.intrinsicHeight = bmp.height
+          this.intrinsicWidth = bmp.width   
+        }
+
+        this.images[seqIndex] = bmp
+
+        if (draw) {
+          this.#paintCanvas(bmp)        
+        }
+
+      })
+      .catch( (err) => {
+        console.error(err)
+      })
+  }
```

Comment:
linting suggestions...
```suggestion
  /**
   * Fetches `url`, converts it into a blob and stores the image data, optionally drawing to the canvas
   * @function #fetchImage
   * @private
   * @param {String} url - image URL to fetch
   * @param {Number} seqIndex - index to store this image 
   * @param {Boolean} draw - whether to draw after fetching
   */
  #fetchImage(url, seqIndex, draw) {
    const request = new Request(url)
    
    fetch(request)
      .then((response) => response.blob())
      .then((blob) => window.createImageBitmap(blob))
      .then((bmp) => {
        if (this.intrinsicHeight === 0) {
          this.intrinsicHeight = bmp.height
          this.intrinsicWidth = bmp.width   
        }
        this.images[seqIndex] = bmp
        if (draw) {
          this.#paintCanvas(bmp)
        }
      })
      .catch((error) => {
        console.error(error)
      })
  }
```

Reply:
@mphstudios I changed this code substantially-enough since this lint tweak that the github tool won't say so, but I *believe* these lint updates are all incorporated into the branch now. `npm run lint` passes on this branch and separately I've issued #965 which quiets the other linting noise that was masking the lint errors in this branch.

Reply:
Erm.. Or, nevermind I see what I missed in these comments -- added and resolved